x-pack/test/screenshot_creation/apps/response_ops_docs/stack_alerting/index_threshold_rule.ts

```diff
@@ -0,0 +1,98 @@
+  const testSubjects = getService('testSubjects');
+  const pageObjects = getPageObjects(['common', 'header']);
+  const screenshotDirectories = ['response_ops_docs', 'stack_alerting'];
+
+  describe('index threshold rule', function () {
+    it('create rule screenshot', async () => {
+      await pageObjects.common.navigateToApp('triggersActions');
+      await pageObjects.header.waitUntilLoadingHasFinished();
+      await rules.common.clickCreateAlertButton();
+      await testSubjects.setValue('ruleNameInput', 'kibana sites - high egress');
+      await testSubjects.click('tagsComboBox');
+      await testSubjects.setValue('tagsComboBox', 'sample-data');
+      await testSubjects.click('solutionsFilterButton');
+      await testSubjects.click('solutionstackAlertsFilterOption');
+      await testSubjects.setValue('solutionsFilterButton', 'solutionstackAlertsFilterOption');
+      await commonScreenshots.takeScreenshot(
+        'rule-types-index-threshold-select',
+        screenshotDirectories,
+        1400,
+        1024
+      );
+
+      await testSubjects.click('.index-threshold-SelectOption');
+      await commonScreenshots.takeScreenshot(
+        'rule-types-index-threshold-conditions',
+        screenshotDirectories,
+        1400,
+        1024
+      );
+
+      await testSubjects.scrollIntoView('selectIndexExpression');
+      await testSubjects.click('selectIndexExpression');
+      const indexComboBox = await find.byCssSelector('#indexSelectSearchBox');
+      await indexComboBox.click();
+      await indexComboBox.type('kibana_sample_data_logs ');
+      const filterSelectItem = await find.byCssSelector(`.euiFilterSelectItem`);
+      await filterSelectItem.click();
+      await testSubjects.click('thresholdAlertTimeFieldSelect');
+      await testSubjects.setValue('thresholdAlertTimeFieldSelect', '@timestamp');
+      await commonScreenshots.takeScreenshot(
+        'rule-types-index-threshold-example-index',
+        screenshotDirectories,
+        1400,
+        1024
+      );
+      await testSubjects.click('closePopover');
+
+      await testSubjects.click('whenExpression');
+      await testSubjects.click('whenExpressionSelect');
+      await testSubjects.setValue('whenExpressionSelect', 'sum()');
+      await testSubjects.click('ofExpressionPopover');
+      const ofComboBox = await find.byCssSelector('#ofField');
+      await ofComboBox.click();
+      await commonScreenshots.takeScreenshot(
+        'rule-types-index-threshold-example-aggregation',
+        screenshotDirectories,
+        1400,
+        1024
+      );
+      await ofComboBox.type('bytes');
+      const ofOptionsString = await comboBox.getOptionsList('availablefieldsOptionsComboBox');
+      const ofOptions = ofOptionsString.trim().split('\n');
+      expect(ofOptions.length > 0).to.be(true);
+      await comboBox.set('availablefieldsOptionsComboBox', ofOptions[0]);
+
+      await testSubjects.click('groupByExpression');
```

Comment:
>     await testSubjects.click('groupByExpression');

@ymao1 Do you have advice on how to close this popover at the end of this group? I can't proceed to the "Define the condition" fields until it's closed, but `await testSubjects.click('closePopover');` doesn't seem to work for this popover.

Reply:
This popover requires 2 clicks to get out of it so in other tests, we seem to hack it by clicking on another element on the form. So after you take the screenshot,

```
// need this two out of popup clicks to close them
const nameInput = await testSubjects.find('ruleNameInput');
await nameInput.click();
```

Reply:
That worked and allowed me to automate some more of the subsequent screenshots in this page. Thanks!